lib/physical/location.rb

```diff
@@ -98,7 +98,7 @@ def to_hash
     end
 
     def ==(other)
-      to_hash == other.to_hash
+      to_hash == other&.to_hash
```

Comment:
We probably should add a type check here as well.

Reply:
Done!